### PR TITLE
Make our homepage conditional on the new dashboard

### PR DESCRIPTION
**Pull request by agjohnson**
This page is not needed anymore, with our website. This page is unused
in our new dashboard already. Instead, redirect right to the log in
page.

Tested on both dashboards

### CODE REVIEW

File: readthedocs/core/views/__init__.py [hunk 1]
```diff
@@ -27,14 +28,27 @@ class HealthCheckView(CDNCacheControlMixin, View):
...
 
-    def get(self, request, *args, **kwargs):
+    def get(self, request, *_, **__):
         return JsonResponse({'status': 200}, status=200)
 
 
 class HomepageView(TemplateView):
 
+    """
+    Conditionally show the home page or redirevt to the login page.
```
Comment (humitos) [inline readthedocs/core/views/__init__.py:38]:
```suggestion
    Conditionally show the home page or redirect to the login page.
```

I can see the new keyboard here 😄

File: readthedocs/core/views/__init__.py [hunk 2]
```diff
@@ -27,14 +28,27 @@ class HealthCheckView(CDNCacheControlMixin, View):
...
+
+    On the current dashboard, this shows the application homepage. Howeber, we
```
Comment (humitos) [inline readthedocs/core/views/__init__.py:40]:
```suggestion
    On the current dashboard, this shows the application homepage. However, we
```

**Review by humitos — APPROVED** [review]

**Comment by agjohnson** [comment]
Docs are failing, but I'm not sure why. This failure is a good case of where I'd love to have the ANSI coloring on the output, so we could see the warnings better.

**Comment by humitos** [comment]
```
/home/docs/checkouts/readthedocs.org/user_builds/docs/checkouts/10189/docs/user/guides/jupyter.rst:115: WARNING: undefined label: 'ipywidgets:/embedding.md#embedding-widgets-in-html-web-pages'
/home/docs/checkouts/readthedocs.org/user_builds/docs/checkouts/10189/docs/user/guides/jupyter.rst:133: WARNING: undefined label: 'ipywidgets:/examples/widget list.ipynb#intslider'
```

I think this is solved in another PR, tho.

**Comment by ericholscher** [comment]
Will this migrate .org to about.?

**Pull request merged**